src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -167,6 +168,25 @@ public boolean validate() {
         mandatory.removeAll(requested);
         missingMandatoryKeys.addAll(mandatory);
 
+        //null element in the values of all the possible keys with DataType as ARRAY
+        final Map<String, Object> contentAsMap = XContentHelper.convertToMap(this.content, false, XContentType.JSON).v2();
```

Comment:
There is a call to `DefaultObjectMapper.readTree()` on line 133. Can't it be reused here?

Reply:
Thanks Vlad, I used this in my new commit